src/Libraries/CoreNodesUI/Input/BasicInteractive.cs

```diff
@@ -22,7 +22,6 @@ public T Value
                     this.value = value;
                     if (!Equals(value, null))
                     {
-                        ForceReExecuteOfNode = true;
```

Comment:
Should this not be an `OR` with `ExecutionHint.GenerateAst` or event `ForceExecute`?